tests/device/poly_kybd_cmd_test.py

```
@@ -413,23 +413,61 @@ def test_get_dynamic_buffer_partial_failure_returns_false(self):
 
 class TestSendOverlayMapping(unittest.TestCase, LockCheckMixin):
 
-    def test_single_chunk_padded_with_noop_keys(self):
+    def test_single_chunk_uses_sized_command_and_fills_the_report(self):
         keeb, device = make_keeb(auto_ack=True)
         ok, msg = keeb.send_overlay_mapping({1: 100, 2: 200})
```

Ruff (0.16.0)

[warning] Unpacked variable `msg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)